rasa/core/policies/policy.py

```diff
@@ -147,18 +151,25 @@ def create(
         """Creates a new untrained policy (see parent class for full docstring)."""
         return cls(config, model_storage, resource, execution_context)
 
-    @classmethod
-    def _create_featurizer(cls, policy_config: Dict[Text, Any]) -> TrackerFeaturizer:
+    def _create_featurizer(self, policy_config: Dict[Text, Any]) -> TrackerFeaturizer:
         policy_config = copy.deepcopy(policy_config)
 
         featurizer_configs = policy_config.get("featurizer")
 
         if not featurizer_configs:
-            return cls._standard_featurizer()
+            return self._standard_featurizer()
+
+        if (
+            POLICY_MAX_HISTORY not in featurizer_configs[0]
+            and POLICY_MAX_HISTORY in policy_config
+        ):
+            featurizer_configs[0][POLICY_MAX_HISTORY] = policy_config[
```

Comment:
I don't think all `TrackerFeaturizer` support this (namely `FullDialogueTrackerFeaturizer` )